docs/elastic-agent-gke-autopilot.md

```diff
@@ -0,0 +1,237 @@
+
+Kubernetes observability is supported through both Elastic Agent installation scenarios ([managed](https://www.elastic.co/guide/en/fleet/current/install-fleet-managed-elastic-agent.html) and [standalone](https://www.elastic.co/guide/en/fleet/current/install-standalone-elastic-agent.html)).
+
+The Kubenretes Integration is an additional package to be installed in your Elastic Cloud and will enhance user experience for Kubenrtes Observability by installing aditonal assets in our Elastic stack.
+
+- Add [Kubernetes Integration](https://docs.elastic.co/integrations/kubernetes)
+  
+![Cloudid](./images/1.png)
+
+- Use default Values and click `Save and Continue`
+
+![Integration](./images/2.png)
+
+- Choose Elastic Agent Installation:
+  - a) For Managed Elastic Agent choose `Add Elastic Agent to your Hosts` . Once butoon cliked, the required  `FLEET_URL` and `FLEET_ENROLLMENT_TOKEN` will be provided for later usage inside Elastic Managed Manifest.
+  - b) For Stanadlone Elastic Agent choose `Add Elastic Agent Later`
+
+![Agent](./images/3.png)
+
+**Use Manifests of next Step 4 to complete installation**
+
+4. **Step 4: Edit provided Agent Manifest**
+
+Depending on the Elastc agent mode chosen, pick relevant manifest below:
+
+1. [Elastic Agent Managed Manifest](./manifests/elastic-agent-managed-kubernetes_gke_autopilot_redused.yaml)
+
+- Set Container Image version of Elastic Agent
+
+```yaml
+containers:
+    - name: elastic-agent-standalone
+      image: docker.elastic.co/beats/elastic-agent:8.5.3
+```
+
+- Provide Elastic Cloud Credentials (In managed scenario you only neeed `FLEET_URL` and `FLEET_ENROLLMENT_TOKEN`)
+  
+```yaml
+- name: FLEET_URL
+  value: "https://123456.fleet.us-central1.gcp.cloud.es.io:443"
+  # Elasticsearch API key used to enroll Elastic Agents in Fleet (https://www.elastic.co/guide/en/fleet/current/fleet-enrollment-tokens.html#fleet-enrollment-tokens)
+  # If FLEET_ENROLLMENT_TOKEN is empty then KIBANA_HOST, KIBANA_FLEET_USERNAME, KIBANA_FLEET_PASSWORD are needed
+  - name: FLEET_ENROLLMENT_TOKEN
+  value: "WmpQbUVJVUJnX0kE6aDg0OGVtNG1TTDJqWlM2VGp"
+  ```
+
+2. [Elastic Agent Stanadlone Manifest](./manifests/elastic-agent-standalone_gke_autopilot_redused.yaml)
+
+- Set Container Image version of Elastic Agent
+
+```yaml
+containers:
+    - name: elastic-agent-standalone
+      image: docker.elastic.co/beats/elastic-agent:8.5.3
+```
+
+- Provide Elastic Cloud Credentials
+  
+```yaml
+# This user needs the privileges required to publish events to Elasticsearch.
+- name: ES_USERNAME
+  value: "elastic"
+# The basic authentication password used to connect to Elasticsearch
+- name: ES_PASSWORD
+  value: "changeme"
+# The Elasticsearch host to communicate with
+- name: ES_HOST
+  value: "https://<url>.kb.us-central1.gcp.cloud.es.io:9243"
+- name: NODE_NAME
+```
+
+5. **Step 5: Install Elastic Agent Manifest:**
+
+For example:
+
+```bash=
+>kubectl apply -f elastic-agent-managed-kubernetes_gke_autopilot_redused.yaml
```

Comment:
Is this `_redused` suffix actually meaningful at this point?

Reply:
Hmm you are right, not so much. I will remove them